src/styles.css

```diff
@@ -176,7 +176,13 @@ h2 {
   background: #B03F2C;
   color: #F2F5F2;
 }
-
+.in-cookbook-state {
```

Comment:
The CSS rule that styles the button when the recipe has been selected and is 'in' the cookbook.